portfolio.css

```diff
@@ -211,3 +207,731 @@ body {
 .btn:disabled {
   cursor: none;
 }
+
+.work {
+  position: absolute;
+  width: 331px;
+  height: 2972px;
+  left: 0;
+  gap: 24px;
+  top: 708px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.recent-works {
+  width: 327px;
+  height: 97px;
+  padding-top: 22px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+}
+
+.indicator_horizontal {
+  width: 48px;
+  height: 4px;
+  border-radius: 24px;
+}
+
+.main-content-cards {
+  display: grid;
+  grid-row: 1 / 7;
+  row-gap: 24px;
+  padding-top: 62px;
+  align-items: center;
+  padding-left: 64px;
+}
+
+.card-work-1 {
+  display: flex;
+  flex-direction: column;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.image {
+  width: 359px;
+  height: 252px;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+  padding: 16px;
+  width: 327px;
+  height: 222px;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.header-work-1 {
+  width: 295px;
+  height: 57px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: #3a4a42;
+  margin: 0;
+  padding-top: 16px;
+}
+
+.list-languages {
+  width: 327px;
+  height: 32px;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  padding: 0;
+  align-items: center;
+  margin: 0;
+}
+
+.ruby {
+  width: 120px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  list-style: none;
+}
+
+.css {
+  width: 45px;
+  padding: 8px 12px;
```

Comment:
- [ ] As requested by the [previous code reviewer, please remove all widths for the technologies list. It is best practice for responsive web design not to use static widths. You can remove the widths and use padding to create spacing in the box.

- [ ] As requested by the previous code reviewer, kindly adjust the contents of your work and about section. The contents seem to be overflowing to the right. Kindly fix that, please.

Figma template*
![Screenshot 2022-08-20 204757](https://user-images.githubusercontent.com/94551484/185763882-8ace61fb-abd2-4022-b77e-619f8d04c459.png)

*your work*
![Screenshot 2022-08-20 204724](https://user-images.githubusercontent.com/94551484/185763884-c0452b5d-6bbf-4740-8d30-1f5e4ee333aa.png)